geniza/footnotes/models.py

```diff
@@ -0,0 +1,68 @@
+from django.db import models
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.contenttypes.fields import GenericForeignKey
+
+from multiselectfield import MultiSelectField
+# https://pypi.org/project/django-multiselectfield/
+
+from geniza.people.models import Person
+
+class SourceType(models.Model):
+    type = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.type
+
+
+class Source(models.Model):
+    # TODO: Multiple authors
+    author = models.ForeignKey(Person, on_delete=models.CASCADE)
+    title = models.CharField(max_length=255)
+    year = models.PositiveIntegerField(blank=True, null=True)
+    # TODO: `edition_number` isn't a number. Can we call it `edition` ?
+    edition_number = models.CharField(max_length=255, blank=True)
```

Comment:
just edition makes sense to me; we can always give it a different verbose name if they want it labeled with more/different detail